### ISSUE TITLE
Strategies for Pre-training Graph Neural Networks

### ISSUE THREAD

**Issue by sanxing-chen** [STayinloves/paper-note#8]
Authors: Weihua Hu, Bowen Liu, Joseph Gomes, Marinka Zitnik, Percy Liang, Vijay Pande, Jure Leskovec
Time: 29 May 2019
Note:
- Naive pre-training GNNs can lead to _negative transfer_ results.
- The strategy is to train a GNN at both the node-level and the graph-level.
  - node-level
    - mask property prediction
    - context prediction
  - graph-level
    - graph-level property prediction
    - graph structural similarity prediction
